ray-operator/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN go mod download
 COPY main.go main.go
 COPY apis/ apis/
 COPY controllers/ controllers/
+COPY pkg/ pkg/
```

Comment:
this also includes pkg/client, which I think is fine? I can also update to `pkg/features`

Reply:
What's the difference in image size with and without `pkg/`?

Reply:
I'll just update this to only copy `pkg/features`, we can always change it to `pkg/` when we introduce more packages that need to be compiled

Reply:
u pdated